frontend/src/pages/ExperimentDetails.test.tsx

```diff
@@ -507,6 +519,145 @@ describe('ExperimentDetails', () => {
     }
   });
 
+  it('enables Archive button when at least one run is selected', async () => {
+    await mockNRuns(4);
+
+    tree = TestUtils.mountWithRouter(<ExperimentDetails {...generateProps()} />);
+    await TestUtils.flushPromises();
+    tree.update();
+
+    for (let i = 0; i < 4; i++) {
+      const archiveButton = (tree.state('runListToolbarProps') as ToolbarProps).actions[
+        ButtonKeys.ARCHIVE
+      ];
+      if (i === 0) {
+        expect(archiveButton!.disabled).toBe(true);
```

Comment:
Use `toBeFalsey` / `toBeTruthy` in these cases?

Reply:
Thank you for pointing it out! Applied.